modules/bcf/config/locales/en.yml

```diff
@@ -28,9 +31,29 @@ en:
         button_perform_import: 'Confirm import'
         button_proceed: 'Proceed'
         button_back_to_list: 'Back to list'
+        no_permission_to_add_members: 'You do not have sufficient permissions to add them as members to the project.'
+        contact_project_admin: 'Contact your project admin to add them as members and start this import again.'
+        continue_anyways: 'Do you want to proceed and finish the import anyways?'
         description: "Provide a BCF-XML v2.1 file to import into this project. You can examine its contents before performing the import."
+        invalid_types_found: 'Invalid topic type names found'
+        invalid_statuses_found: 'Invalid status names found'
+        invalid_priorities_found: 'Invalid priority names found'
+        invalid_emails_found: 'Invalid email addresses found'
+        unknown_emails_found: 'Unknown email addresses found'
+        non_members_found: 'Non project members found'
+        import_types_as: 'Import them all setting type to'
```

Comment:
This formulation sounds very strange to me. I had a look a translator and seems correct. However maybe we could simple use `Import all types as`. What do you think?

Reply:
Hmm, I agree that the current string look awkward. What I want to communicate is

- You overwrite those that are listed and not the others. Therefore I use the word "them".
- I wanted to repeat the word "type" in the sentence as this is probably the only line of text that you read when you have already imported several times and do not read the heading anymore.

For now I change it to "Import all these types as". If you or others dislike it, we can change it anytime.

Reply:
👍 That is fine for me